app.py

```diff
@@ -37,6 +38,18 @@ def new_channel_callback(**payload) -> None:
     send_new_channel_message(web_client, 'admin', new_channel_name)
 
 
+@slack.RTMClient.run_on(event='team_join')
+def new_user_callback(**payload) -> None:
+    """Catch team_join event.
+
+    Triggered when a user has joined the workspace.
+    """
+    web_client = payload['web_client']
+    new_user = payload['data']['user']
+
+    send_new_user_message(web_client, 'admin_logs', new_user)
```

Comment:
At some point do we want to start abstracting this out into at least constants somewhere if not an environment configuration?

Reply:
Absolutely. Let's write a ticket for this.